contracts/RootChain/RootChain.sol

```diff
@@ -260,18 +266,23 @@ contract RootChain {
 
             // prevent a potential DoS attack if from someone purposely reverting a payment
             uint256 amountToAdd = currentExit.amount.add(minExitBond);
+
+            // if the amount we want to send is greater than the contract's balance - the amount
+            // allocated for invalid sends, terminate the function.
+            if (amountToAdd >= this.balance - totalWithdrawBalance) {
```

Comment:
Should this be amountToAdd > this.balance - totalWithdrawBalance. Otherwise wouldn't there always be be some eth you couldn't get from the contract.

Reply:
Good catch. Fixed and pushed.